### PR TITLE
Add React/Vue snippets for Viz Drill-Down Chart topics

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md [hunk 1]
```diff
@@ -97,6 +97,83 @@ The main idea is to filter the data source by the `parentID` for different drill
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart ...
+            :data-source="dxDataSource">
```
Comment (AlexJameson) [inline concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md]: Why use the 'dx' prefix? I suggest removing it and use 'dataSource' here and below

Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md]: It's case for using DevExpress.data.DataSource. I'll fix other cases

Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md]: Fixed

File: concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md [hunk 2]
```diff
@@ -159,4 +236,88 @@ The main idea is to filter the data source by the `parentID` for different drill
...
+    import DataSource from 'devextreme/data/data_source';
+
+    const population = [
+        // ...
+    ];
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+             this.state = {
```
Comment (AlexJameson) [inline concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md]: I suggest removing state here as in [this case](https://github.com/DevExpress/devextreme-documentation/pull/751#discussion_r424449261).

Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/91 Drill-Down Chart/01 Provide Data.md]: Fixed

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**